cli/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ _Released 1/16/2024 (PENDING)_
 - Updated [`@cypress/unique-selector`](https://www.npmjs.com/package/@cypress/unique-selector) to include a performance optimization. It's possible this could improve performance of the selector playground. Addressed in [#28571](https://github.com/cypress-io/cypress/pull/28571).
 - Replace [`CircularJSON`](https://www.npmjs.com/package/circular-json) with its successor [`flatted`](https://www.npmjs.com/package/flatted) version `3.2.9`. This resolves decoding issues observed in complex objects sent from the browser. Addressed in [#28683](https://github.com/cypress-io/cypress/pull/28683).
 - Updated [`better-sqlite3`](https://www.npmjs.com/package/better-sqlite3) from `8.7.0` to `9.2.2` to fix macOS Catalina issues. Addresses [#28697](https://github.com/cypress-io/cypress/issues/28697).
+- Refreshed dependencies of [dockerode](https://www.npmjs.com/package/dockerode) to update [cpu-features](https://www.npmjs.com/package/cpu-features) from `0.0.2` to `0.0.9`. This resolves an issue building Cypress from source on Windows operating systems. Addresses [#28725](https://github.com/cypress-io/cypress/issues/28725).
```

Comment:
Since this is a development dependency update, no changelog entry is added and the PR title can be listed as `chore`.

Reply:
Thanks for your advice! I've removed the changelog entry and updated the PR title / commit to use `chore` instead of `fix`.